src/TranslationServiceProvider.php

```diff
@@ -127,4 +136,33 @@ protected function getConfigFile(): string
             DIRECTORY_SEPARATOR . 'config' .
             DIRECTORY_SEPARATOR . 'translation.php';
     }
+
+    /**
+     * @return string
+     */
+    protected function getViewsPath(): string
+    {
+        return __DIR__ .
+            DIRECTORY_SEPARATOR . '..' .
+            DIRECTORY_SEPARATOR . 'resources' .
+            DIRECTORY_SEPARATOR . 'views';
+    }
+
+    private function bootBladeExtensions()
```

Comment:
🙏 Separate nested `foreach` in order to make the code readable and much clean.

Reply:
Should be fixed